ipmi/warewulf-ipmi.spec.in

```diff
@@ -13,8 +13,16 @@ URL: http://warewulf.lbl.gov/
 Source: %{name}-%{version}.tar.gz
 ExclusiveOS: linux
 Requires: warewulf-common %{name}-initramfs-%{_arch} = %{version}-%{release}
+
+%if 0%{?rhel} >= 8 || 0%{?sle_version} >= 150000
+BuildRequires: ipmitool
+Requires: ipmitool
+%define CONF_FLAGS "--with-local-ipmitool=yes"
+%endif
+
 BuildRequires: warewulf-common openssl-devel
-Conflicts: warewulf < 3
+BuildRequires: autoconf, automake
+Conflicts: warewulf < 3.9
```

Comment:
Curious why this conflicts with < 3.9?

Reply:
I started adjusting during my initial scrub, thinking there was a virtual warewulf package in one of the specs. After I realized warewulf 2 was issued as a package, it made sense and I thought I reverted these to 3. In any case, I'll issue a quick commit to fix.